integration/sawtooth_integration/docker/test_poet_liveness.yaml

```diff
@@ -53,6 +53,19 @@ services:
         /project/sawtooth-core/signing:\
         /project/sawtooth-core/cli"
 
+  smallbank-workload:
+    image: sawtooth-dev-python:$ISOLATION_ID
+    volumes:
+      - $SAWTOOTH_CORE:/project/sawtooth-core
+    expose:
+      - 8008
+    command: "bash -c \"\
+      sawtooth keygen smallbank-key; \
+      while true; do curl -s http://rest-api-0:8008/state | grep -q head; if [ $$? -eq 0 ]; then break; fi; sleep 0.5; done; \
+      /project/sawtooth-core/perf/smallbank_workload/bin/smallbank-workload load --key /root/.sawtooth/keys/smallbank-key.priv --rate 1 --target http://rest-api-0:8008,http://rest-api-1:8008,http://rest-api-2:8008,http://rest-api-3:8008,http://rest-api-4:8008 \
```

Comment:
I would suggest making the parameters for transactions per batch explicit, since we may change the defaults.

Reply:
I'll add that in a separate PR.